tests/pkg/tenant/tenant_setup.go

```diff
@@ -8,8 +8,9 @@ type TenantStatus string
 type TenantType string
 
 const (
-	testProvider      = "Compass Tests"
-	testDefaultTenant = "Test Default"
+	testProvider                = "Compass Tests"
+	testDefaultTenant           = "Test Default"
+	testDefaultSubaccountTenant = "Test Default Subaccount"
```

Comment:
Most probably the new tenant will be inserted by the K8s "tenant-loader" job but I think it doesn't execute every time? As far as I remember only when the director image/component is changed because the job uses the director image. Which in that case will do the job for us. I'm mentioning it to monitor the job and see if the new tenant will be inserted into our environments(dev-main, dev)